src/services/token/index.ts

```diff
@@ -0,0 +1,79 @@
+import type { DeleteResult } from 'typeorm';
+import jwt from 'jsonwebtoken';
+import { dbInstanse } from '../db/utils/getConnection';
+import Token from '../db/entities/Token';
+import User from '../db/entities/User';
+
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+require('dotenv').config();
+
+type GenerateTokens = {
+  accessToken: string;
+  refreshToken: string;
+};
+
+class TokenService {
+  static generateTokens = async (payload: any): Promise<GenerateTokens> => {
+    const accessToken = jwt.sign(
+      payload,
+      String(process.env.JWT_ACCESS_SECRET),
+      {
+        expiresIn: '30m',
+      },
+    );
+    const refreshToken = jwt.sign(
+      payload,
+      String(process.env.JWT_REFRESH_SECRET),
+      {
+        expiresIn: '30d',
+      },
+    );
+
+    return {
+      accessToken,
+      refreshToken,
+    };
```

Comment:
i suggest to change to `return {tokens: {access, refresh}}` it's more readable and clearly delineates the entities